editquality/codegen/generate.py

```diff
@@ -1,5 +1,10 @@
 import jinja2
 
+ALG_FILE_NAME = {
```

Comment:
This should be moved to editquality/config.py

Reply:
I'm not sure I agree.  Writing into the config is a huge source of confusion for me right now.  I'm not sure that is a good pattern.

Reply:
It could be a new file if you wish, but the must-have in my opinion is that anything specific to the editquality domain needs to live outside of the `codegen` package.

Reply:
AHa!  Makes sense.  Can do.

Reply:
We should chat about the config.py file, I agree that it's confusing.  Maybe as simple as renaming it to parameters.py and writing a better docstring?